Makefile issues

## General Information

- This is a: bug report
- Captain version: `main`
- OS: Debian 10 x64 LXC

## Issue Report

Makefile is causing two errors when trying to install the DEB package from source. The first is:

```
rm ./pkg/deb/usr/local/bin/.keep
rm: cannot remove './pkg/deb/usr/local/bin/.keep': No such file or directory
make: *** [Makefile:28: captain-atc.deb] Error 1
```

The second issue is an extra `$(MAKE) deb` that got left behind.

## Related issues

Probably caused by #143 and #149

--- a/ATC/Makefile
+++ b/ATC/Makefile
@@ -15,18 +15,16 @@ test: $(NAME)
 	go test -v -coverprofile=cover.out ./... 2>&1 | go-junit-report > report.xml
 
 clean:
-	rm ./$(NAME)
-	rm ./pkg/*.deb
-	rm ./*.db
+	rm ./$(NAME) || true
+	rm ./pkg/*.deb || true
+	rm ./*.db || true
 
 install: $(NAME).deb
-	$(MAKE) deb
 	sudo apt install ./pkg/captain-atc_*_amd64.deb
 
 $(NAME).deb: $(NAME)
-	$(MAKE) build
 	cp $(NAME) ./pkg/deb/usr/local/bin/$(NAME)
-	rm ./pkg/deb/usr/local/bin/.keep
+	rm ./pkg/deb/usr/local/bin/.keep || true
 	dpkg-deb --build ./pkg/deb
 	mv ./pkg/deb.deb ./pkg/captain-atc_$(VERSION)_amd64.deb
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Radar: ATC API URL is now configurable in a new `config.yaml`.
+- ATC: Fixed issue where DEB packages couldn't be built from source.
 
 ### Security
 

--- a/Radar/Makefile
+++ b/Radar/Makefile
@@ -12,9 +12,9 @@ test: $(NAME)
 
 $(NAME).deb: $(NAME)
 	cp $(NAME) ./pkg/deb/usr/local/bin/$(NAME)
-	rm -R ./pkg/deb/etc/captain/radar/templates/
+	rm -R ./pkg/deb/etc/captain/radar/templates/ || true
 	cp -R ./templates/ ./pkg/deb/etc/captain/radar/templates/
-	rm ./pkg/deb/usr/local/bin/.keep
+	rm ./pkg/deb/usr/local/bin/.keep || true
 	dpkg-deb --build ./pkg/deb
 	mv ./pkg/deb.deb ./pkg/captain-radar_$(VERSION)_amd64.deb
 
@@ -24,7 +24,7 @@ install: $(NAME).deb
 all: install
 
 clean:
-	rm $(NAME)
-	rm ./pkg/*.deb
+	rm $(NAME) || true
+	rm ./pkg/*.deb || true
 
 .PHONY: clean all install test build

